contracts/modules/capital/Pool.sol

```diff
@@ -127,8 +129,9 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
   ///
   function getPoolValueInEth() public override view returns (uint) {
 
-    uint total = address(this).balance + swapValue;
+    uint total = address(this).balance;
     uint assetCount = assets.length;
+    uint _assetsInSwapOperatorBitmap = assetsInSwapOperatorBitmap;
```

Comment:
assetsInSwapOperatorBitmap is uint32 should _assetsInSwapOperatorBitmap be uint32 as well?